common/src/main/java/io/netty/util/internal/SocketUtils.java

```diff
@@ -42,9 +43,26 @@
  */
 public final class SocketUtils {
 
+    private static Enumeration<Object> EMPTY = new Enumeration<Object>() {
```

Comment:
Could be more concise as:
```java
EMPTY = Collections.enumeration(Collections.emptyList());
```

Reply:
good idea (also missed the "final" here).